docker/README.md

```diff
@@ -12,29 +12,62 @@ Steps:
 Command example:
 
     docker build -t infrasim-compute .
-## docker.sh
-The bash script includes below functions:
- * docker.sh can help you setup the docker running environment;
- * pull an infrasim-compute image from docker hub
- * run an infrasim-compute in a container
+
+## docker.py
+This script is used to manage infrasim-compute docker containers. It has 3 sub commands, start, stop and show.
+The command of 'start' starts new infrasim nodes and the usage like,
+    usage: docker.py start [-h] [-a] -i ETH -n NODES [NODES ...] [-l]
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      -a, --append          append new infrasim nodes
+      -i ETH, --if ETH      network interface for bridge connection
+      -n NODES [NODES ...], --nodes NODES [NODES ...]
+                            node configuration, <yml_file,count>
+      -l, --list            generate an IP list of containers afterafter creation
+                            done. 
+
+This function helps you setup the docker running environment by,
+ * pull an infrasim-compute image from docker hub if need,
  * Setup the container/host network to connect to a dhcp network interfaces. We leverage
  *OpenVswitch* and *pipework* to setup the dhcp network.
-
-## cleanup.sh
-This script is used to remove all infrasim-compute docker containers and the openvswitch "ovs-br0",
-which is set up in docker.sh.
+ * configure infrasim-compute and run it in a container
+ * give an IP list of containers for remote access.
+ 
  
+                            
+The command of 'stop' stops infrasim nodes and clean up the environment.
+
+    usage: docker.py stop [-h] [-i ETH]
+
+    optional arguments:
+      -h, --help        show this help message and exit
+      -i ETH, --if ETH  the network interface
+      
+It stop all containers related to specified network interface and restore the network of host. 
+If no eth is specifiled, it clean all containers.
+                  
+The command of 'show" gives an IP list of containers for remote access by SSH
+
+    usage: docker.py show [-h] [-i ETH]
+
+    optional arguments:
+      -h, --help        show this help message and exit
+      -i ETH, --if ETH  the network interface
+
+
 Network connection as picture below:
 
 ![container_network](https://github.com/InfraSIM/tools/blob/master/docker/Infrasim-compute_container_network.jpg)
```

Comment:
Regarding the picture, can we add another containerized node into network? This will better illustrate how multiple nodes are connected together.